meetings/agenda-minutes-2020-07-23.md

```diff
@@ -0,0 +1,153 @@
+July 23, 2020
+
+Attendees:
+
+* Shane Carr (SFC)
+
+* Jason Williams (JWS)
+
+* Justin Grant (JGT)
+
+Agenda:
+
+* Translate the survey in order to get more responses from non-Gregorian-calendar developers? (idea from Yulia)
+
+    * SFC: It might be good to just ask people we know.
+
+    * JGT: I assume Japanese would be good. There’s a large Arabic software development community, but not sure how many aren’t also fluent in English.
+
+    * SFC: Even if fluent, good to translate to drive interest.
+
+    * JWS: So 1. Do we have resources to do it and 2. Does it have value?
+
+    * JWS: We could make an issue for this?
+
+    * Jason to make issue for translations of the survey
+
+* How to approach IETF calext/calconnect working group?
+
+    * For [#702](https://github.com/tc39/proposal-temporal/issues/702) RFC 5545 vs DST questions for IETF calendar standards ("calext") group
+
+    * JGT: How is CalConnect related to calext ([https://datatracker.ietf.org/wg/calext/email/](https://datatracker.ietf.org/wg/calext/email/))
+
+    * SFC: CalConnect is iCal i'm not as familiar with calext.
+
+    * JGT: Who will answer my Qs about the RFC 5545 spec?
+
+    * JGT: Will reach out to CalConnect
+
+* Duration balancing
```

Comment:
Same, I'd say delete the points if not discussed

Reply:
Done.